aptos-move/framework/aptos-framework/sources/TokenV1.move

```diff
@@ -698,6 +702,38 @@ module AptosFramework::TokenV1 {
         }
     }
 
+    public(script) fun burn_script(
```

Comment:
Shouldn't have `script` in the public entrypoint name.

Reply:
I think we need to have a naming convention.
Since there are duplicated names, to differentiate the `public(script) fun`, `public fun` and `fun`, I have been use xxx_script, xxx, xxx_internal to differentiate them. what is the naming convention we should have then ?

Reply:
FYI, `(script)` is deprecated recently, and `entry` is used now. https://github.com/move-language/move/pull/186